app/presenters/bill-runs/two-part-tariff/amend-billable-returns.presenter.js

```diff
@@ -38,10 +37,24 @@ function go (billRun, reviewChargeElement, licenceId) {
         reviewChargeElement.reviewChargeReference.reviewChargeVersion.chargePeriodEndDate
       )
     },
-    licenceId
+    licenceId,
+    authorisedQuantity: _authorisedQuantity(reviewChargeElement)
   }
 }
 
+/**
+ * The user can only enter a volume on the billable returns that is less than the authorised volume. The authorised
+ * volume is either the authorised volume on the charge element or the authorised volume on the charge reference.
+ * Whichever is lower.
+ */
+function _authorisedQuantity (reviewChargeElement) {
+  const { chargeElement, reviewChargeReference } = reviewChargeElement
+  const chargeElementQuantity = chargeElement.authorisedAnnualQuantity
+  const chargeReferenceQuantity = reviewChargeReference.chargeReference.volume
+
+  return chargeElementQuantity > chargeReferenceQuantity ? chargeReferenceQuantity : chargeElementQuantity
```

Comment:
Personally I'd favor using `Math.min` which we have done in a few other files, and would do `return Math.min(chargeElement.authorisedAnnualQuantity, reviewChargeReference.chargeReference.volume` and get rid of lines 52 & 53.

I suspect you may disagree from chats we have had previously,  in which case wonder what @Cruikshanks thinks?